### ISSUE TITLE
AERCalc AIR install -- updates needed

### ISSUE THREAD

**Issue by RDmitchell** [LBNL-ETA/AERCalc#28]
AERCalc 1.3.07 (the latest version) with WINDOW 7.8.12 and THERM 7.8.04 (which are not the latest versions of those programs)

Things which need to be updated before we can put a version on our public website
* update sample database in the W7 subdirectory to be WINDOW 7.8.12
* Update WINDOW and THERM versions to be
  * WINDOW 7.8.12
    * (1/15/2021 RM): I did this in the following directory (I just changed the w7.exe file to be 7.8.12, may want to revert that and have Stephen do it in a more organized way
    * G:\Shared drives\AERCalc\Versions\V 1_3 Versions\1.3.7\AERCalc AIR Install Files-W7-8_12\AERCalc AIR Install Files - 1.3.7\W7
  * THERM 7.8.06

Keep adding anything else here as we discover it

**Comment by RDmitchell** [comment]
Closing - no longer relevant
